spec/services/record_declarations/retained/early_career_teacher_spec.rb

```diff
@@ -16,7 +16,7 @@
       declaration_type: "retained-1",
       course_identifier: "ecf-induction",
       lead_provider_from_token: another_lead_provider,
-      evidence_held: "Test evidence",
+      evidence_held: %w[training-event-attended self-study-material-completed other].sample,
```

Comment:
Do we really want to do more "sample" based testing? Wouldn't it be better to test them all rather than risk a random failure that would be harder to diagnose?